stripe/src/test/java/com/stripe/android/view/MaskedCardViewTest.java

```diff
@@ -0,0 +1,248 @@
+            "},\n"+
+            "\"receiver\": {\n"+
+            "\"address\": \"test_1MBhWS3uv4ynCfQXF3xQjJkzFPukr4K56N\",\n"+
+            "\"amount_charged\": 0,\n"+
+            "\"amount_received\": 0,\n"+
+            "\"amount_returned\": 0\n"+
+            "},\n"+
+            "\"status\": \"pending\",\n"+
+            "\"type\": \"card\",\n"+
+            "\"usage\": \"single_use\",\n"+
+            "\"card\": " + EXAMPLE_JSON_SOURCE_CARD_DATA + "\n"+
+            "}";
+
+    private static final String EXAMPLE_JSON_SOURCE_BITCOIN = "{\n"+
+            "\"id\": \"src_19t3xKBZqEXluyI4uz2dxAfQ\",\n"+
+            "\"object\": \"source\",\n"+
+            "\"amount\": 1000,\n"+
+            "\"client_secret\": \"src_client_secret_of43INi1HteJwXVe3djAUosN\",\n"+
+            "\"created\": 1488499654,\n"+
+            "\"currency\": \"usd\",\n"+
+            "\"flow\": \"receiver\",\n"+
+            "\"livemode\": false,\n"+
+            "\"metadata\": {\n"+
+            "},\n"+
+            "\"owner\": {\n"+
+            "\"address\": null,\n"+
+            "\"email\": \"jenny.rosen@example.com\",\n"+
+            "\"name\": \"Jenny Rosen\",\n"+
+            "\"phone\": \"4158675309\",\n"+
+            "\"verified_address\": null,\n"+
+            "\"verified_email\": null,\n"+
+            "\"verified_name\": null,\n"+
+            "\"verified_phone\": null\n"+
+            "},\n"+
+            "\"receiver\": {\n"+
+            "\"address\": \"test_1MBhWS3uv4ynCfQXF3xQjJkzFPukr4K56N\",\n"+
+            "\"amount_charged\": 0,\n"+
+            "\"amount_received\": 0,\n"+
+            "\"amount_returned\": 0\n"+
+            "},\n"+
+            "\"status\": \"pending\",\n"+
+            "\"type\": \"bitcoin\",\n"+
+            "\"usage\": \"single_use\"\n"+
+            "}";
+
+    static final String JSON_CARD = "{\n" +
+            "    \"id\": \"card_189fi32eZvKYlo2CHK8NPRME\",\n" +
+            "    \"object\": \"card\",\n" +
+            "    \"address_city\": \"Des Moines\",\n" +
+            "    \"address_country\": \"US\",\n" +
+            "    \"address_line1\": \"123 Any Street\",\n" +
+            "    \"address_line1_check\": \"unavailable\",\n" +
+            "    \"address_line2\": \"456\",\n" +
+            "    \"address_state\": \"IA\",\n" +
+            "    \"address_zip\": \"50305\",\n" +
+            "    \"address_zip_check\": \"unavailable\",\n" +
+            "    \"brand\": \"MasterCard\",\n" +
+            "    \"country\": \"US\",\n" +
+            "    \"currency\": \"usd\",\n" +
+            "    \"customer\": \"customer77\",\n" +
+            "    \"cvc_check\": \"unavailable\",\n" +
+            "    \"exp_month\": 8,\n" +
+            "    \"exp_year\": 2017,\n" +
+            "    \"funding\": \"credit\",\n" +
+            "    \"fingerprint\": \"abc123\",\n" +
+            "    \"last4\": \"5555\",\n" +
+            "    \"name\": \"John Cardholder\"\n" +
+            "  }";
```

Comment:
I think that can be done -- i'll look into it, because I end up having to pull those in a lot.